force-app/main/default/pages/STG_PanelDeceasedBatch.page

```diff
@@ -0,0 +1,49 @@
+<apex:page controller="STG_PanelDeceasedBatch_CTRL" docType="html-5.0" standardStylesheets="false" >
```

Comment:
Not more Visualforce! 😉  This is a simple enough page that I think it's appropriate for our timeline, but my goodness we need to update these Settings...

Reply:
We briefly discussed lwc, but I'm not sure that makes sense with every one of the other pages handled this way...